src/helm/proxy/clients/ai21_client.py

```diff
@@ -103,14 +111,14 @@ def parse_token(raw: Dict, first: bool) -> Token:
             text_length: int = raw["textRange"]["end"] - raw["textRange"]["start"]
             # "topTokens" can be None when sending a request with topKReturn=0
             top_logprobs: Dict[str, float] = dict(
-                (fix_text(x["token"], first), x["logprob"]) for x in raw["topTokens"] or []
+                (fix_text(x["token"], first), x["raw_logprob"]) for x in raw["topTokens"] or []
```

Comment:
Should we make a comment in the code what `raw_logprob` is and when it was added?